README.md

```diff
@@ -1 +1,44 @@
-# todo
\ No newline at end of file
+![Hireable](https://cdn.rawgit.com/hiendv/hireable/master/styles/default/yes.svg)
+
+  <h1 align="center">Todo App</h1>
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Kindly let's fix the screenshot path because it seems to be broken for now 👍